api/src/main/java/org/jboss/forge/roaster/model/Type.java

```diff
@@ -20,8 +20,12 @@
 
    String getName();
 
+   String getSimpleName();
+
```

Comment:
getName() just strips generics, getSimpleName preserves generics, but strips qualifications from names based on imports.
Happy to rename to clarify  : something like getSimpleNameByImports?
I have added javadocs to clarify the behavior